action.yml

```diff
@@ -0,0 +1,59 @@
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+inputs:
+  repo:
+    description: "The repo to run against"
+    required: false
+  instance_count:
+    description: "The number of instances to create"
+    required: false
+    default: '1'
```

Comment:
confused... I thought the whole point of the previous PR was to make it so that you don't need this?

Reply:
That's not the case. The previous PR allows for us to specify more than one instance, but it requires you to set it. This is not functionality that existed in the original project.